client/components/data/query-rewind-backup-status/index.js

```diff
@@ -11,19 +11,32 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import Interval, { EVERY_SECOND } from 'lib/interval';
-import { getRewindBackupProgress as getBackupProgress } from 'state/activity-log/actions';
+import { getRewindBackupProgress } from 'state/activity-log/actions';
 
 class QueryRewindBackupStatus extends Component {
 	static propTypes = {
 		downloadId: PropTypes.number,
 		siteId: PropTypes.number.isRequired,
 	};
 
+	componentWillMount() {
+		// We want to run this only once: when the page is loaded. In such case, there is not known download Id.
+		// If there's a download Id here it means this was mounted during an action requesting progress for a
+		// specific download Id, so we will do nothing here,since it will be handled by the <Interval /> below.
```

Comment:
This comment needs updating since we took away the download id

Reply:
I won't remove this since it serves as a distinction between a query initiated on page load vs a continuous one when the progress card is shown.